Kajabi/sage-lib
# Data Card - added Rails component and conditional title tag

## Description

- [x] - created Rails component
- [x] - updated doc props
- [x] - a11y - added conditional `title_tag`

## Test notes

No visual changes

### Steps for testing
1. Visit rails data card page: http://localhost:4000/pages/object/data_card
2. Verify the updated docs, demo, and props
